students/meet_shah/lesson02/series.py

```diff
@@ -0,0 +1,32 @@
+
+def fib(n):
```

Comment:
Cool, looks good. We also wanted a sum_series function, but this look like you've good the idea. Check out the solution we did in class if that was causing you issues.

Reply:
yep missed the sum_series :(